src/Http/Response.php

```diff
@@ -1069,6 +1069,16 @@ public function type($contentType = null)
     }
 
     /**
+     * Returns the current content type.
+     *
+     * @return string
+     */
+    public function getType()
```

Comment:
Should we update the deprecation notice for `Response::type()`

Reply:
I do it in new PR after merging this 
+
implementing `public function withMimeType(array $mimeTypes)`

Reply:
also https://github.com/cakephp/cakephp/pull/11336#issuecomment-337341305

Reply:
If someone wants to use a content type with no short alias, they should give the full content type. I don't know if we need to allow the short type aliases to be modified.

Reply:
Anyway we need a method for adding new mimetypes before full deprecating `type()`

Reply:
@saeideng You already can using `withHeader('Content-Type', 'foo/bar')` :)

Reply:
>withHeader('Content-Type', 'foo/bar')

@ADmad 
so we no longer need to new method( `withMimeType`) :)

Reply:
We could add that convenience too if people think it's worth saving typing of `'Content-Type'` :)